### PR TITLE
Lets dchat-controlled mobs fire guns

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Allows dchat-controlled mobs to fire guns.
Also adds handling for nodrop/abstract items, so admins can equip their victims with whatever they see fit.

## Why It's Good For The Game
I got a request for this, and it provides one more little thing for dchat-controllable mobs.

## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/89928798/692b1894-4862-496a-acd3-8c489900fa47

## Testing
See video.

## Changelog
:cl:
tweak: A new option, "shoot", is available when deadchat controlling a human to fire a gun it might be holding.
/:cl:

### CODE REVIEW

**Comment by matttheficus** [comment]
image

**Comment by Probablyjay19** [comment]
This time on DEAD CHAT PLAYS, a naked goober with a laser rifle movers around da station and shoots people 😎

File: code/modules/mob/living/carbon/human/human_mob.dm [hunk 1]
```diff
@@ -2156,6 +2164,28 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
...
+
+	var/obj/item/gun/held_gun = get_active_hand()
+	if(!held_gun)
+		visible_message("<span class='warning'>[src] makes fingerguns towards [possible_target]!</span>")
+	else if(!istype(held_gun))
+		visible_message("<span class='warning'>[src] points [held_gun] towards [possible_target]!</span>")
+	else
+		// for his neutral special, he wields a Gun
+		held_gun.afterattack(possible_target, src)
+		visible_message("<span class='danger'>[src] fires [held_gun][isturf(possible_target) ? "" : " towards [possible_target]!"]</span>")
```
Comment (S34NW) [inline code/modules/mob/living/carbon/human/human_mob.dm:2188]:
```suggestion
	if(!held_gun)
		visible_message("<span class='warning'>[src] makes fingerguns towards [possible_target]!</span>")
		return
	if(!istype(held_gun))
		visible_message("<span class='warning'>[src] points [held_gun] towards [possible_target]!</span>")
		return
	// for his neutral special, he wields a Gun
	held_gun.afterattack(possible_target, src)
	visible_message("<span class='danger'>[src] fires [held_gun][isturf(possible_target) ? "" : " towards [possible_target]!"]</span>")
```

**Review by S34NW — COMMENTED** [review]
sniper monkey

**Review by S34NW — APPROVED** [review]

**Pull request merged**